pkg/controller/wildflyserver/transaction_recovery.go

```diff
@@ -80,8 +80,9 @@ func (r *ReconcileWildFlyServer) checkRecovery(reqLogger logr.Logger, scaleDownP
 		// The pod was already searched for the recovery port, marking that into the annotations
 		annotations := wfly.MapMerge(
 			scaleDownPod.GetAnnotations(), map[string]string{markerRecoveryPort: strconv.FormatInt(int64(scaleDownPodRecoveryPort), 10)})
+		patch := client.MergeFrom(scaleDownPod.DeepCopyObject())
```

Comment:
For me to understand. This line is to get origin value of `scaleDownPod`. Then we change `scaleDownPod` to desired value and pass both values to Patch method.

Maybe I would use different naming of `patch variable`, but I see it is aligned with previous reviewed fix [1] So probably it is OK.

[1] https://github.com/wildfly/wildfly-operator/pull/227/files